src/cmd-kola

```diff
@@ -66,6 +67,10 @@ if ignition_version == "2.2.0":
 if os.getuid() != 0 and not ('-p' in args.subargs):
     kolaargs.extend(['-p', 'qemu-unpriv'])
 
+# tpm devices for qemu not supported on s390x/ppc
+if platform.machine() == "s390x" or platform.machine() == "ppc64le":
```

Comment:
How about `if cmdlib.basearch() in ["s390x", "ppc64le"]:` ?

Reply:
yes..much..cleaner...done..